tests/fv3_conf/ccpp_control_run.IN

```diff
@@ -56,5 +56,3 @@ cp    @[RTPWD]/WW3_input_data/mod_def.* .
 cp    @[RTPWD]/WW3_input_data/@[SYEAR]@[SMONTH]@[SDAY]/ww3_multi.inp .
 cp    @[RTPWD]/WW3_input_data/@[SYEAR]@[SMONTH]@[SDAY]/rmp_src* .
 fi
-
-cp ${PATHRT}/../FV3/ccpp/suites/suite_${CCPP_SUITE}.xml suite_${CCPP_SUITE}.xml
```

Comment:
So do we not need  the ccpp suite file in the run directory?

Reply:
Not needed, correct.